pinot-core/src/main/java/org/apache/pinot/core/common/MinionConstants.java

```diff
@@ -72,6 +72,11 @@ private MinionConstants() {
    */
   public static final int DEFAULT_MAX_ATTEMPTS_PER_TASK = 1;
 
+  /**
+   * Segment download thread pool size to be set at task level.
+   */
+  public static final String SEGMENT_DOWNLOAD_THREAD_POOL_SIZE = "segmentDownloadThreadPoolSize";
```

Comment:
Consider `segmentDownloadParallelism` which is easier for users to understand IMO